src/main/java/blusunrize/lib/manual/gui/ManualScreen.java

```diff
@@ -55,7 +55,14 @@
 	private ClickableList entryList;
 	private ClickableList suggestionList;
 
+	private final boolean doLastActive;
```

Comment:
I would have called it something different here (`setAsLastActive` or similar), but this works as well.

Reply:
I agree, naming here is a bit awkward.